include/core.h

```diff
@@ -65,8 +65,8 @@ void trackingFrame2Frame(const cv::Mat& projMatrl,
                          std::vector<cv::Point2f>& pointsLeft_t0,
                          std::vector<cv::Point2f>& pointsLeft_t1,
                          cv::Mat& points3D_t0,
-                         cv::Mat& rotation,
-                         cv::Mat& translation);
+                         cv::Matx<double, 3, 3>& rotation,
```

Comment:
why double?

Reply:
It was declared as double before
`cv::Mat rotation = cv::Mat::eye(3, 3, CV_64F);`
`cv::Mat translation_stereo = cv::Mat::zeros(3, 1, CV_64F);`